cloud/gcp/gke.md

```diff
@@ -25,6 +25,8 @@ Support for GKE is available only in portworx release version 1.4 and above.
 
 Portworx is supported on GKE cluster provisioned on [Ubuntu Node Images](https://cloud.google.com/kubernetes-engine/docs/node-images). So it is important to specify the node image as **Ubuntu** when creating clusters.
 
+To manage GCP disks, Portworx needs access to the GCP Compute Engine API. By default, all instances in a GKE cluster have these permissions. If you modify the permissions, ensure that the service account for the instances has atleast **Read Write** access to the **Compute Engine** API. This can be configured in the "Project Access" section when creating the GKE cluster.
```

Comment:
If the GKE cluster itself needs all permissions, then this is a moot issue. I though you mentioned someone was able to change the default permissions of the cluster. Hence the above note mentions that Portworx atleast needs Read Write to the Compute Engine.

Reply:
Last time I created the GKE cluster with default permissions, it had access. This is because for the GCP cloud provider for k8s, it needs all of the same permissions. 

While creating the cluster, one can optionally go to  "Project access" section and change these default permissions to a more restrictive scope.

Reply:
![kubernetes engine portworx eng](https://user-images.githubusercontent.com/25230810/42185024-4e9e3394-7dfc-11e8-987b-ef9f9188f314.png)
Attached image of the default that is selected when creating a GKE cluster. That only has read permission for storage.
All instances have permission to _make_ GCP APIs, but not write access to storage APIs